src/services/account-service.js

```diff
@@ -0,0 +1,22 @@
+import { StorageService } from "./storage";
+
+export class AccountService {
+  static add(account) {
+    return AccountService.get().then(accounts => {
+      if (!accounts) {
+        AccountService.set([account]);
+      } else {
+        let updatedAccounts = [account].concat(accounts);
+        AccountService.set(updatedAccounts);
```

Comment:
`this.set`

Reply:
also you could just do `this.set([account, ...accounts])` -- es6 array rest-spread